mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule Macfly.MixProject do
   defp deps do
     [
       {:msgpax, "~> 2.3"},
-      {:httpoison, "~> 1.8"},
+      {:httpoison, "~> 1.8", only: :test},
```

Comment:
@jeregrine Does this still prevent it from being a dependency for ui-ex?

Reply:
yes